### PR TITLE
Fix license parsing

### PR DESCRIPTION
**Pull request by mobasherul**

### CODE REVIEW

File: agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java [hunk 1]
```diff
@@ -248,7 +248,7 @@ public boolean installVoter(InstanceId instanceId, TerracottaVoter terracottaVot
   }
 
   public boolean installClusterTool(InstanceId instanceId, String hostName, Distribution distribution,
-                                    License license, String kitInstallationName, SecurityRootDirectory securityRootDirectory,
+                                    String kitInstallationName, License license, SecurityRootDirectory securityRootDirectory,
```
Comment (mathieucarbou) [inline agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java:251]: do we really need to change the parameter ordering ?

Comment (mobasherul) [reply agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java:251]: No I will fix it

File: client-internal/src/main/java/org/terracotta/angela/client/ConfigTool.java [hunk 2]
```diff
@@ -308,11 +308,6 @@ public ConfigTool activate() {
       clusterName = instanceId.toString();
     }
     List<String> args = new ArrayList<>(Arrays.asList("activate", "-n", clusterName, "-s", terracottaServer.getHostPort()));
-    String licensePath = tsa.licensePath(terracottaServer);
-    if (licensePath != null) {
-      args.add("-l");
-      args.add(licensePath);
-    }
```
Comment (mathieucarbou) [inline client-internal/src/main/java/org/terracotta/angela/client/ConfigTool.java:315]: 👍

File: common/src/main/java/org/terracotta/angela/common/TerracottaServerInstance.java [hunk 3]
```diff
@@ -71,7 +71,7 @@ public TerracottaServerInstance(TerracottaServer terracottaServer, File kitDir,
     this.workingDir = workingDir;
     this.distribution = distribution;
     this.portAllocator = portAllocator;
-    this.licenseFileLocation = license == null ? null : new File(kitDir, license.getFilename());
+    this.licenseFileLocation = license == null ? null : new File(workingDir, license.getFilename());
```
Comment (mathieucarbou) [inline common/src/main/java/org/terracotta/angela/common/TerracottaServerInstance.java:74]: 👍

**Review by mathieucarbou — APPROVED** [review]

**Pull request merged**